sources/packages/web/src/views/aest/CASInvoices.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <full-page-container :full-width="true">
+    <template #header
+      ><header-navigator
+        title="Corporate Accounting Services"
```

Comment:
I know this is initial version of UI, but should we call a page as `Corporate Accounting Services`? Question to business.

Reply:
Business was ok with the current naming at this moment, the preliminary UI was shared on different occasions.
We can wait for further feedback from the final users.